runtime-modules/forum/src/lib.rs

```diff
@@ -1038,21 +1034,32 @@ decl_module! {
         }
 
         /// Edit post text
+        // TODO need a safer approach for frame_system call
```

Comment:
Why was this comment added here? I do not understand it, also the heading is "EDit post text" , which does not match the extrinsic.

Reply:
Just moved it from the inner function, performing actual mutation. Seems, that that naming mismatch was here before, i can fix.